examples/stamping/myapp.py

```diff
@@ -16,14 +16,15 @@
    # The shell service is used to run the example.
     (window2)$ celery -A myapp shell
 
-   # Use (copy) the content of shell.py to run the workflow via the
+   # Use (copy) the content of the examples modules to run the workflow via the
    # shell service.
 
-   # Use one of two demo runs via the shell service:
+   # Use one of demo runs via the shell service:
```

Comment:
```suggestion
   # Use one of the 3 demo runs via the shell service:
```

Reply:
The `examples.py` has 3 demos, the revoke example has 2 demos, so I don't want to commit to a number in the docs and just say "one of the demo runs"